File: week-1---cplusplus-intro/lab_age.py
import sys

def main():
	print('What is your age?')
	age = int(input())

	if age < 5:
		print("You're a baby")
	elif age >= 13 and age <= 18:
		print("You're a teenager")

	if age >= 16:
		print("You can drive")
	
	if age >= 18:
		print("You can vote")

if __name__ == '__main__':
	sys.exit(main())

File: week-1---cplusplus-intro/lab_age.cpp
#include <iostream>

int main()
{
    int age;
	std::cout << "What is your age?" << std::endl;
	std::cin >> age;
    if (age<5)
    {
      std::cout << "You're a baby" << std::endl;
    }
    else if (age >= 13 && age <= 18)
    {
      std:: cout << "You're a teenager" << std::endl;
    }
    if (age>=16)
    {
      std::cout << "You can drive" << std::endl;
    }
    if (age>=18)
    {
      std::cout << "You can vote" << std::endl;
    }

    return 0;
}